apps/src/aiComponentLibrary/chatMessage/chat-message.module.scss

```diff
@@ -77,6 +77,11 @@ $assistant-color: $light_gray_100;
   justify-content: flex-end;
 }
 
+.userProfaneMessageButton {
+  width: 120px;
+  margin-top: -10px;
```

Comment:
What's this need for?

Reply:
The button that hides/shows the profane user message.

Reply:
Oh yep, was wondering specifically why we needed the negative margin? Is it to cut into the chat message margin?

Reply:
oh yeah - so that the button was a little closer to its associate chat message.

Reply:
Ah interesting - I don't think the chat message itself has any extra margin, so I'm wondering if it might be better to nest the button inside the `container` div and add a bit of padding/margin/gap there instead? I'm just thinking that the -10px makes some assumptions about how chat messages are spaced out which may potentially differ from app to app?

Reply:
Yeah - I tried that at first, but then the button and message were set in a row.
<img width="493" alt="Screenshot 2024-08-21 at 11 10 06 AM" src="https://github.com/user-attachments/assets/01fbf178-898a-4a4a-b798-ef8c60d26c9c">

So then I tried updating `flex-direction` to `column`, but then it messed with the user chat message styling (should be aligned to the right but after style update the message takes up entire width).
<img width="517" alt="Screenshot 2024-08-21 at 11 11 11 AM" src="https://github.com/user-attachments/assets/d7dbec10-7c34-4ec4-b9e6-c14cfc3f2f10">

Here's what it should look like:
<img width="505" alt="Screenshot 2024-08-21 at 11 12 02 AM" src="https://github.com/user-attachments/assets/bac1b2f9-bf8e-4e7e-9a1c-80ffc402e649">


I think I'll leave as-is for now and we can address in a follow-up if you have ideas to resolve this. Thanks!